src/psyclone/psyGen.py

```diff
@@ -5967,10 +5967,27 @@ class CodeBlock(Node):
     :param statements: list of fparser2 AST nodes representing the Fortran \
                        code constituting the code block.
     :type statements: list of :py:class:`fparser.two.utils.Base`
+    :param structure: optional argument indicating whether this code \
```

Comment:
I made it optional as I thought it might not be something that makes sense for a CodeBlock in a different language. Do you think it is a generic property?

Reply:
OK, I've made the argument mandatory.